jobs/tendrl-cluster-test-jobs.yml

```diff
@@ -195,10 +195,7 @@
           done
           # prepare env
           ssh usmqe@${{qe_server}} "export PATH=\$PATH:\$HOME/.local/bin:\$HOME/bin; \
```

Comment:
I was wondering if this `PATH` setup is necessary, because when I switch to usmqe user from root shell via `su - usmqe`, I see this already defined:

```
[usmqe@mbukatov-qe-server-centos7 ~]$ echo $PATH
/opt/rh/rh-python35/root/usr/bin:/usr/local/bin:/bin:/usr/bin:/usr/local/sbin:/usr/sbin:/home/usmqe/.local/bin:/home/usmqe/bin
```

But it seems to be omitted in plain login shell:

```
$ ssh usmqe@mbukatov-qe-server-centos7 'echo $PATH'
/opt/rh/rh-python35/root/usr/bin:/usr/local/bin:/usr/bin
```

So ok.